Submissions/Yolo86/Task_1/csvtojson.py

```diff
@@ -0,0 +1,14 @@
+import csv
+import json
+
+with open('C:\\Users\\SAKSHAM\\Desktop\\OpenCode20\\2-Python t1\\Monty_Python_Class\\Task_1\\movies.csv','r', encoding="utf8") as csv_file:
+	fieldnames = ("movieId","title","genres")
+	csv_reader = csv.DictReader(csv_file,fieldnames)
+
+	with open('csv2json.json','w', encoding="utf8") as json_file:
+
+		for line in csv_reader:
+
+			if line['genres'] == 'Comedy':
```

Comment:
There can be movies whose genre contains other genres also apart from only 'Comedy' such as 'Drama' or 'Action' so make your code more flexible.

Reply:
Alright..I'll make the required changes